packages/utils/src/lib/report.unit.test.ts

```diff
@@ -182,3 +185,86 @@ describe('loadReport', () => {
     ).rejects.toThrow('validation');
   });
 });
+
+describe('audits sorting', () => {
```

Comment:
It would be great to split testing the sort based on individual entity and priority and minimise the number of items in each case to showcase it in the simplest way.

Reply:
Added unit tests for both `sortAudits()` and `sortCategoryAudits()`, covered all conditions.